crates/sui-core/src/authority_aggregator.rs

```diff
@@ -1397,7 +1399,11 @@ where
                 let ct_digest = ct.digest();
                 debug!(?ct, ?ct_bytes, ?ct_digest, "Collected tx certificate");
                 Ok(Some(ProcessTransactionResult::Certified(
-                    ct.verify(&self.committee)?,
+                    // We do not verify the user sig here, because we know we are starting out with
+                    // a VerifiedTransaction. Because transaction may use a stateful authenticator
+                    // (such as zklogin) it is possible (though unlikely) that we may not be able
+                    // to replicate the verification decision that was made by the authorities.
+                    ct.verify_committee_sigs_only(&self.committee, original_tx)?,
```

Comment:
to confirm, this is the only place we skip user sign verify right?

Reply:
there is one other place I just added in SafeClient. By the way, we don't actually skip verification here - the idea of providing the original tx is that we are proving that we already have a VerifiedTransaction in hand. (Yes, there are places where we do `VerifiedTransaction::new_unchecked` but the putative safety of that operation should flow through to here.)